<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
  
   <rosparam file="$(find ORB-SLAM2_DENSE)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find ORB-SLAM2_DENSE)/config/costmap_common_params.yaml" command="load" ns = "local_costmap" />
    <rosparam file="$(find ORB-SLAM2_DENSE)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find ORB-SLAM2_DENSE)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find ORB-SLAM2_DENSE)/config/base_local_planner_params.yaml" command="load" />
 #   <remap from="odom" to="/orb_slam/odom"/>
    <remap from="/cmd_vel" to="/cmd_vel_mux/input/teleop" />

  </node>
</launch>
